zcash_client_sqlite/src/wallet/init/migrations.rs

```diff
@@ -131,8 +132,66 @@ pub(super) fn all_migrations<P: consensus::Parameters + 'static>(
     ]
 }
 
+/// All states of the migration DAG that have been exposed in a public crate release, in
+/// the order that crate users would have encountered them.
+///
+/// Omitted versions had the same migration state as the first prior version that is
+/// included.
+#[allow(dead_code)]
+const PUBLIC_MIGRATION_STATES: &[&[Uuid]] = &[
+    V_0_4_0, V_0_6_0, V_0_8_0, V_0_9_0, V_0_10_0, V_0_10_3, V_0_11_0,
+];
+
+/// Leaf migrations in the 0.4.0 release.
+const V_0_4_0: &[Uuid] = &[add_transaction_views::MIGRATION_ID];
+
+/// Leaf migrations in the 0.6.0 release.
+const V_0_6_0: &[Uuid] = &[v_transactions_net::MIGRATION_ID];
```

Comment:
I verified that as of v0.6.0 the ASCII art was correct, and `v_transactions_net` was the only leaf.